walsamlee/politico
# #163416980 Add Login Endpoint

#### What does this PR do?
Add `login` endpoint
#### Description of Task to be completed?
Have user sign in
#### How should this be manually tested?
- Clone the project, https://github.com/walsamlee/politico
- Check out into the proper branch, `ft-login-163416980`
#### What are the relevant pivotal tracker stories?
#163416980
